src/useInterval/useIntervalStories.test.ts

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom';
+import { waitFor, render } from '@muban/testing-library';
+import { Demo } from './useInterval.stories';
+import { timeout } from '../useTimeout/useTimeout.test.utils';
+
+describe('useInterval stories', () => {
+  it('should render', () => {
+    const { getByText } = render(Demo);
+
+    expect(getByText('Test Area')).toBeInTheDocument();
+  });
+
+  it('should start immediate and be called after 100ms', async () => {
+    const { getByText } = render(Demo, { startImmediate: true, interval: 100 });
+
+    await waitFor(() => expect(getByText('interval called')).toBeInTheDocument());
+  });
+
+  it('should start after clicking start and be called after 100ms', async () => {
+    const { getByText, getByRef } = render(Demo, { interval: 100 });
+    const startButton = getByRef('start-button');
+
+    startButton.click();
+
+    await waitFor(() => expect(getByText('interval called')).toBeInTheDocument());
+  });
+
+  it('should stop the interval after starting', async () => {
+    const { getByText, getByRef } = render(Demo, { interval: 100 });
+    const startButton = getByRef('start-button');
+    const stopButton = getByRef('stop-button');
+
+    startButton.click();
+    await timeout(100);
+    stopButton.click();
+
+    await waitFor(() => expect(getByText('interval stopped')).toBeInTheDocument());
```

Comment:
This is testing the functionality of the stop button, not the interval :D

Reply:
Isn't that what the story test functions are for? The same thing appears to be happening in the `useToggleStories` 🤔

Reply:
> The same thing appears to be happening in the useToggleStories 🤔

Not really, there the stop button invokes the hook, and the state of the hook updates the UI.
In this case, the button invokes internal state that is unrelated to the hook.

So either we add something that actually tests the hook functionality, or we should leave it out (this is confusing :D)

Reply:
Ok I assumed we had to write tests for all the story scenarios, but I'll remove it 🤓